src/styles/spacing/index.md.njk

```diff
@@ -127,4 +127,6 @@ If you need to constrain the width of an element independently of the [grid syst
 
 As with the spacing override classes the width override classes start with `govuk-!-`. The second part of the class name signifies the width on larger screen sizes. For example, `govuk-!-width-one-half`  will apply a width of 50% and `govuk-!-width-two-thirds`will apply a width of 66.66%.
 
+These examples are for the generic width override classes. See [more guidance on setting text input width](/components/text-input/#use-appropriately-sized-text-inputs) specifically.
```

Comment:
LGTM but could we change to the below to avoid having one word left over after the link?

```suggestion
These examples are for the generic width override classes - read specific [guidance on setting text input width](/components/text-input/#use-appropriately-sized-text-inputs).
```